src/actions/events.js

```diff
@@ -405,6 +495,34 @@ events.on(FILTER_CLEAR, function() {
   tree.unset(['pages', 'repo', 'filter']);
 });
 
+events.on(BUILD_FILTER, function(event) {
+  const {owner, name, value} = event.data;
+  if (value === '') {
+    tree.unset(['pages', 'repo', owner, name, 'filter']);
+  } else {
+    tree.set(['pages', 'repo', owner, name, 'filter'], value);
+  }
```

Comment:
Scoping by `owner/name` gives ability to track filter values / suggestions per repository so in case one switches to another repo and filters by different attributes and then comes back to the previous one, the filters / filtered builds list will be maintained. 

This is different to filter used to select repo in the left hand sidebar: https://github.com/drone/drone-ui/blob/master/src/actions/events.js#L400